### PR TITLE
[2584] Fix incorrect styling on fee or funding page

### PR DESCRIPTION
**Pull request by davidgisbey**
### Context

The fee or funding page is not in line with the prototype

### Changes proposed in this pull request
- Refactor the fee or funding page be in line with the prototype

## Guidance to review

https://manage-courses-prototype.herokuapp.com/new/V978/funding

### Checklist

- [x] Make sure all information from the Trello card is in here
- [x] Attach to Trello card
- [x] Rebased master
- [x] Cleaned commit history
- [x] Tested by running locally

### CODE REVIEW

**Comment by davidgisbey** [comment]
[Fee paying or salary page: Style fixes](https://trello.com/c/Lq0NC3wd/2584-fee-paying-or-salary-page-style-fixes)

File: app/views/courses/fee_or_salary/_form_fields.html.erb [hunk 1]
```diff
@@ -7,21 +7,11 @@
       <h1 class="govuk-fieldset__heading">
         Is it fee paying or salaried?
       </h1>
+      <br>
```
Comment (timabell) [inline app/views/courses/fee_or_salary/_form_fields.html.erb]: `<br />` shurely

Comment (davidgisbey) [reply app/views/courses/fee_or_salary/_form_fields.html.erb]: image
Guess it depends 🤷‍♂

Comment (timabell) [reply app/views/courses/fee_or_salary/_form_fields.html.erb]: oh okay that was xhtml, it's fine in html5, encouraged in fact

**Comment by timabell** [comment]
Some before/after screenshots in the PR would be nice

**Comment by davidgisbey** [comment]
Thanks 👍

**Pull request merged**